### ISSUE TITLE
Keep scroll position

### ISSUE THREAD

**Issue by syranez** [syranez/tab-reload-on-select#6]
> I find that 1.4.2 has a default compared to 1.3.1.1: it does not keep 
> the scroll position.
> 
> With 1.3.1.1, the tab reloads and I don't have to scroll if was not on 
> top, while with 1.4.2 it goes on top, and I almost hace to scroll to the 
> position I was.
> 
> I use it mainly on a site that looks like:
> 
> where I am scrolled down.
> 
> With the new version, it goes on top:
> 
> 
> 
> and I have to scroll to see the useful content.
> 
> Not a big deal, but having to do it every time gets old rapidly.
> 
> At the moment, I put back m modified 1.3.1.1 and disabled the updates.

**Comment by BobHD** [comment]
Same for me. I was about to open an issue, when I saw that syranez had already done it.

For me, it works with Water fox 2020.10, but only after a complete un-install and re-install with Tabreloadonselect{1.3.1.1}.xpi that I saved before with FEBE. I disabled the updates. And it is a big deal, it's a great loss of time.

**Comment by syranez** [comment]
Fixed in 1.5.0 (released now)

The scroll position gets lost because the addon tried to refresh the page. Now this per default deactivated and can optionally activated by the user.
